packages/scaffolder/docs/2-features.md

```diff
@@ -0,0 +1,289 @@
+  This is used for WordPress plugins that want to automatically place
+  themselves at `wp-content/plugins/<plugin-name>`.
+- `theme`: The destination will be resolved to the WordPress theme directory.
+  This is used for WordPress themes that want to automatically place
+  themselves at `wp-content/themes/<theme-name>`.
+
+The `destination-resolver` can be set in the `config.yml` file of the feature:
+
+```yaml
+name: Case Study
+description: An optional description of the feature.
+config:
+  destination-resolver: plugin
+```
+
+#### Defining Features Without a Subdirectory
+
+Features can also be defined without a subdirectory on the `features` key in the
+project's `.scaffolder/config.yml` file.
+
+```yaml
+features:
+  - name: Case Study
+    description: An optional description of the feature.
+    inputs:
+      - name: caseStudyName
+        description: "Case Study Name"
+        type: string
+      - name: tests
+        description: "Include Tests?"
+        type: boolean
+        default: true
+    files:
+      - source: case-study.stub
+        destination: src/case-study/{{ wpClassFilename inputs.caseStudyName }}
+      - source: case-study-feed.stub
+        destination: src/feeds/{{ wpClassFilename inputs.caseStudyName }}.php
+      - source: test.stub
+        if: "{{ inputs.tests }}"
+        destination: tests/Features/{{ psrClassFilename inputs.caseStudyName prefix="" suffix="" }}
+```
+
+Subdirectories are **strongly recommended** to keep the project organized, but
+features can be defined in the `.scaffolder/config.yml` file if desired.
+
+### Remote Features
+
+Remote Features are features that are not specific to or located within a
+project and can be scaffolded anywhere. Out of the box, the scaffolder comes
+with a set of remote features that can be scaffolded. These can include new
+WordPress plugins, projects, themes, etc. Additional remote feature can be
+sourced from a remote repository or a local directory. In the future, we'll be
+using NPM to manage remote features, too. See
+[Configuration](./5-configuration.md) for more information.
+
+Remote Features follow the same syntax as local features, but are not located
+within a project.
+
+## Feature Inputs
+
+Inputs are a list of values that the user will be prompted before generation.
+Inputs are optional but can be used to customize the feature to the user's
+needs. Inputs are defined as a list of objects with the following properties:
+
+- `name`: Required. The name of the input. This is used to identify the input in the list
+  of inputs.
+- `description`: Optional. The description of the input. This is used to describe the
+  input to the user.
+- `type`: Optional. The type of the input. This is used to validate the input. The
+  following types are supported:
+    - `string`: A string value.
+    - `boolean`: A boolean value. The input will be a checkbox.
+    - `select`: (Not yet supported!) A select value. The input will be a select
```

Comment:
imo we should keep not yet supported items out of the README - I'd rather add the docs to the readme when support is added